docs/examples/contact-manager.md

```diff
@@ -117,16 +117,16 @@ class Contact(Skill):
 This part involves listing the contacts that have been added to the database. We will be doing this by getting the data from the database the same as how we did for the adding to the database which is `c.execute("SELECT * FROM contacts")`. With the data we can use a for loop to go through all the things in the database. It should look something like this:
 
 ```python
-    @match_regex(r"Show contacts")     
-    async def showContacts(self, message):
-        c.execute("SELECT * FROM contacts")
-        rows = c.fetchall()
-        if len(rows) == 0:
-            await message.respond('No Contacts added, Please Use Command "Add contact: PhoneNumber, Name, Surname, Email"')
-        else: 
-            for row in rows:
-                phoneNumber, name, surname, email = row
-                await message.respond(f'Phone Number: {phoneNumber}, Name: {name}, Surname: {surname} and Email: {email}')
+@match_regex(r"Show contacts")     
+ async def showContacts(self, message):
+  c.execute("SELECT * FROM contacts")
+  rows = c.fetchall()
+  if len(rows) == 0:
+    await message.respond('No Contacts added, Please Use Command "Add contact: PhoneNumber, Name, Surname, Email"')
+  else: 
+     for row in rows:
+        phoneNumber, name, surname, email = row
+        await message.respond(f'Phone Number: {phoneNumber}, Name: {name}, Surname: {surname} and Email: {email}')
```

Comment:
The indentation here is still incorrect. Please ensure each level of indentation contains 4 spaces.